test/powershell/Modules/Microsoft.PowerShell.Utility/FormatHex.Tests.ps1

```diff
@@ -415,14 +327,34 @@ Describe "FormatHex" -tags "CI" {
             }
         )
 
-        foreach ($testCase in $testCases)
-        {
-            RunExpectedErrorTestCase $testCase
+        It "<Name>" -Skip:$($skipTest -or (-not $certProviderAvailable)) -TestCase $testCases {
+
+            param ($Name, $PathParameterErrorCase, $Path, $InputObject, $InputObjectErrorCase, $ExpectedFullyQualifiedErrorId)
+
+            try
+            {
+                if ($PathParameterErrorCase)
+                {
+                    $result = Format-Hex -Path $Path -ErrorAction Stop
+                }
+                if ($InputObjectErrorCase)
+                {
+                    $result = Format-Hex -InputObject $InputObject -ErrorAction Stop
+                }
+            }
+            catch
+            {
+                $thrownError = $_
+            }
+
+            $thrownError.FullyQualifiedErrorId | Should Match $ExpectedFullyQualifiedErrorId
         }
     }
 
     Context "Continues to Process Valid Paths" {
 
+        $skipTest = ([System.Management.Automation.Platform]::IsLinux -or [System.Management.Automation.Platform]::IsOSX)
```

Comment:
I'd be happier if this was combined with the construct in 378
IsLinux -or IsOSX -or (-not $certProviderAvailable)
then 378 could just be:
`It "<Name>" -Skip:$skip`
same with the code below